cli/001-cli-improve-piping-support.md

```diff
@@ -0,0 +1,33 @@
+- Start Date: 2021-06-11
+- RFC PR:
+- Issue:
+
+### Support better piping of output 
+Problem: currently piping the output into other tools is not well supported. For example utilities such as `grep`, `wc`, `jq` may work with very limited set of commands / in limited functionality. 
+
+Structure outputs so it would be easy to pipe it into common utilities. 
+
+To achieve that:
+
+ - `wc`, `more` support: integrate output with `less` cmd by default if it's available in the OS. If `less` is not found, create and use builtin functiniality to output all available rows immediately (similar to how `cat` behaves).
+ 
+ Examples: 
+ ``` bash
+ tctl workflow list --open --all | wc
```

Comment:
I'm inferring from this that the default format for non-TTY would not be table (otherwise `wc` would count the header/divider rows). If that's the case it should be specified here or a link given to the proposal which changes that behaviour. Also I think this should be `wc -l`?

Reply:
I wouldn't change the output for tty vs non-tty, it provides a strange experience.
Think of what would happen to a user that runs `tctl wf list ...`, realizes they want to pipe the results and get different output.

Reply:
```suggestion
 tctl workflow list --open --all | wc -l
```